index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h1>Hello world we are software developers!</h1>
+    <h1 id="title"> Survey Form </h1>
+    <p id="description"> This is a survey form for microverse trials</p>
+
+    <form id ="survey-form">
+      <div>
+        <label id="name-label" for="name" > Name <input  id="name" type="text" name="name" placeholder="Type your name here" required > </label> 
```

Comment:
- [ ] Please remove the id tag from all label and input as they are not useful in this project